cla_public/translations/cy/LC_MESSAGES/messages.po

```diff
@@ -1548,7 +1548,7 @@ msgid "Read more about cookies on ‘Check if you can get legal aid’"
 msgstr "Darllenwch fwy am ddefnyddio cwcis ar ‘Gwirio os ydych yn gymwys i gael cymorth cyfreithiol’"
 
 #: cla_public/templates/_cookie-consent-form.html:42
-msgid "Cookies used to improve ‘Check if you can get legal aid’"
```

Comment:
- Will you not need to also change the Welsh translation for this? 
- Will you not need to also change other references e.g. like on line 1547 and 1548